doc/circuitikzmanual.tex

```diff
@@ -1775,13 +1775,16 @@ \subsubsection{Transistor paths}\label{sec:transasbip}
 
 \subsection{Electronic Tubes}
 
+\small{Contributed by J. op den Brouw.}
```

Comment:
Don't you want to add an email? Like Contributed by ... (\texttt{foo@bar.com})? You're call...

Reply:
Yep, that would  be nice. Done.